python/GangaDirac/Lib/Splitters/SplitterUtils.py

```diff
@@ -3,7 +3,7 @@
 from GangaDirac.Lib.Utilities.DiracUtilities import execute
 from GangaDirac.Lib.Backends.DiracUtils import result_ok
 from Ganga.Utility.Config import getConfig
-from GangaDirac.Lib.Files import DiracFile
+from GangaDirac.Lib.Files.DiracFile import DiracFile
```

Comment:
Indeed it is subtle. This is a common problem in Ganga where we have a class called `Foo` inside a file called `Bar/Foo.py`. This is fine until we, in `Bar/__init.py__`, do `from Foo import Foo` and suddenly `Bar.Foo` is ambiguous.